index.js

```diff
@@ -32,7 +32,8 @@ import "./styles.scss";
 // For local testing, set this
 // const AD_DECISION_URL = "http://ethicaladserver:5000/api/v1/decision/";
 const AD_DECISION_URL = "https://server.ethicalads.io/api/v1/decision/";
-const AD_CLIENT_VERSION = 1;
+const AD_CLIENT_VERSION = 1;  // Sent with the ad request
```

Comment:
I'd probably semvar this from the start -- at least `1.0` -- but even better if we could sync it up to the actual value in `package.json`.